pkg/scheduler/core/generic_scheduler.go

```diff
@@ -486,6 +486,17 @@ func (g *genericScheduler) findNodesThatFit(pod *v1.Pod, nodes []*v1.Node) ([]*v
 
 		checkNode := func(i int) {
 			nodeName := g.cache.NodeTree().Next()
+
+			/* Iterate each plugin to verify current node */
```

Comment:
the `//` for comment is dominant in the codebase.

Reply:
Got it